core/src/main/java/org/mapfish/print/output/Values.java

```diff
@@ -102,18 +107,20 @@ public Values() {
      * @param jasperTemplateBuild the directory where the jasper templates are compiled to
      */
     public Values(
+            final String applicationId,
             final String jobId,
             final PJsonObject requestData,
             final Template template,
             final File taskDirectory,
             final MfClientHttpRequestFactoryImpl httpRequestFactory,
             final File jasperTemplateBuild) {
-        this(jobId, requestData, template, taskDirectory, httpRequestFactory, jasperTemplateBuild, null);
```

Comment:
Rather than deleting the constructor without application ID, I would introduce the new one with applicationID and make this one call the new One, using a constant like UNKNONW-APPLICATION-ID

Reply:
I am not sure if that's a good idea, in the application I never pass null, I pass null in the test.
Then is it a good idea to create a constructor only for the test and generate an issue (missing application ID in the related logs) if he is used in the application?